two_hemis.html

```diff
@@ -58,16 +58,20 @@
 		<script>
 			angular.module('navigator', [])
 			.controller('NavigateController', ['$scope', function($scope) {
-				$scope.brain = new Brain("nav-brain", function(mesh) {
-					if (!mesh) {
-						$scope.selectedLabel = "";
-						$('#plot-canvas').innerHtml = "";
-					} else {
-						$scope.selectedLabel = mesh.name;
-						do_hemiplot("plot-canvas", mesh, $scope.brain);
-					}
-					$scope.$apply();
-				}, "lh_files_to_load.json");
+                $scope.brain = new Brain({
+                    divID: "nav-brain",
+                    callback: function(mesh) {
+                        if (!mesh) {
+                            $scope.selectedLabel = "";
+                            $('#plot-canvas').empty();
+                        } else {
+                            $scope.selectedLabel = mesh.name;
+                            do_hemiplot("plot-canvas", mesh);
+                        }
+                        $scope.$apply();
+                    },
+                    manifest: 'lh_files_to_load.json'
+                });
```

Comment:
This is what constructing `Brain` looks like now: pass an object with all properties you want to define.